source/getting-started/index.md

```diff
@@ -37,6 +38,16 @@ installing Homebrew, run `brew install node` to install Node.js.
 Once you've got Node.js installed, re-run `node --version` to verify your
 computer's setup.
 
+If you already have npm and Node.js installed, it's a good idea to make sure you're on the latest verson of npm since it gets updated more frequently than Node.js does. 
+
+To update npm, run:
+
+```bash
+sudo npm install npm -g
+```
+
+For those unfamiliar with npm, the reason why you are using `-g` is to install the tool globally, so you can use it as a command line tool. You will do the same later when installing Ember and Ember CLI. On the other hand, if you wanted to depend on the package from your own module using something like Node's require, then you would want to install it locally without the `-g`.
```

Comment:
"globally", "module", "require"... these are all terms that I don't think we can assume our readers are familiar with. I like the intent of this paragraph, but I think it will end up causing more confusing than clarity.